client/src/partials/development/reports/core.html

```diff
@@ -0,0 +1,73 @@
+<header data-header>
+  {{ReportCtrl.title}}
+</header>
+
+<!-- TODO: If page structure should be replaced with implicit correctness through -->
+<!-- dynamic, database driven routes -->
+<nav ng-if="::ReportCtrl.state.isValid">
+  <button ng-click="ReportCtrl.hookGeneration()" class="btn btn-primary btn-default">Create new {{ReportCtrl.title}} Document</button>
+
+  <div class="pull-right">
+    <button disabled class="btn btn-default">Manage recurring report</button>
+    <button disabled class="btn btn-default">Settings</button>
+  </div>
+</nav>
+
+<main>
+  
+  <!-- Binding one way declutters the digest loop after the initial page load -->
+  <section ng-if="::ReportCtrl.state.isInvalid">
+    <div class="alert alert-warning">
+      Report key '{{ReportCtrl.reportKey}}' is not supported.
+    </div>
+  </section>
+  
+  <section ng-if="::ReportCtrl.state.isValid">
+  
+    <!-- TODO: Review how expensive the conditional templating below is -->
+    <!-- Searching for archives -->
+    <div ng-if="ReportCtrl.state.isLoading" ng- class="panel panel-default">
+      <div class="panel-body">
+        <i style="padding-right : 0.5em;" class="glyphicon glyphicon-search"></i> <strong>Searching archives...</strong>
+      </div>
+    </div>
+
+    <!-- No archive data found -->
+    <div ng-if="ReportCtrl.archivesEmpty" class="alert alert-info">
+      <i style="padding-right : 0.5em;" class="glyphicon glyphicon-folder-open"></i> <strong> No reports found in the archive.</strong> Use the 'Create new {{ReportCtrl.title}} Document' button to create a new report.
+    </div>
+
+    <!-- Found archives -->
+    <div ng-if="ReportCtrl.archivesEmpty===false && ReportCtrl.state.suppressSuccessDisplay===false" class="alert alert-success alert-dismissible">
+      <button ng-click="ReportCtrl.suppressSuccess()" style="line-height : 0.85;" class="close" type="button" aria-label="close"><span aria-hidden="true">&times;</span></button>
+      <i style="padding-right : 0.5em;" class="glyphicon glyphicon-ok"></i> Displaying <strong>{{ReportCtrl.archives.length}}</strong> archived reports.
+    </div>
+
+    <!-- TODO: Replace table with grid pending grid analysis (UI grid/ slick grid) -->
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Report Title</th>
+          <th>Date</th>
+          <th>Tags</th>
+          <th>User</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr ng-repeat="archive in ReportCtrl.archives | orderBy : !archive.created">
+          <td>{{archive.title}}</td>
+          <td>{{archive.created | date}}</td>
+          <td></td>
```

Comment:
I assume tags will be some sort of `<span class="label label-{{tag.color}}" ng-repeat="tag in archive.tags">{{ tag.label }}</span>` here?